README.md

```diff
@@ -2,7 +2,15 @@
 
 [![](https://img.shields.io/crates/v/tao?style=flat-square)](https://crates.io/crates/tao) [![](https://img.shields.io/docsrs/tao?style=flat-square)](https://docs.rs/tao/) ![](https://img.shields.io/crates/l/tao?style=flat-square)
 
-Cross-platform application window creation library in Rust that supports all major platforms like Windows, macOS, Linux, iOS and Android.
+[![Chat Server](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/SpmNs4S)
+[![devto](https://img.shields.io/badge/blog-dev.to-black.svg)](https://dev.to/tauri)
+[![devto](https://img.shields.io/badge/documentation-tauri.studio-purple.svg)](https://tauri.studio/docs/getting-started/intro)
+[![https://good-labs.github.io/greater-good-affirmation/assets/images/badge.svg](https://good-labs.github.io/greater-good-affirmation/assets/images/badge.svg)](https://good-labs.github.io/greater-good-affirmation)
```

Comment:
Image alts don't make a ton of sense here :]

Reply:
true

Reply:
but really just an academic thing. i think once we ship out the codebase to the auditors, we will def have time to circle back and make a cohesive revision to all this stuff